packages/base/src/index.js

```diff
@@ -143,7 +143,7 @@ module.exports = {
     'no-return-assign': ['error', 'except-parens'],
     'no-script-url': 'error',
     'no-self-compare': 'error',
-    'no-shadow': 'error',
+    'no-shadow': ['error', { builtinGlobals: true, hoist: 'functions' }],
```

Comment:
Nit: `hoist: 'functions'` is the default option for `hoist`, so this doesn't seem strictly necessary to include